spyder/plugins/tests/test_projects.py

```diff
@@ -14,6 +14,7 @@
 import pytest
 
 # Local imports
+import spyder.plugins.base
```

Comment:
This is only valid in `master`. For `3.x` you need to write:

    import spyder.plugins

Reply:
Yes, I tested against master, then rebased against 3.x, but forgot to run the tests again. Stupid.